### DISCUSSION TITLE
[Advanced Analytics][time-series chart] Tooltip and column title of DATA for Time comparison with absolution difference is confusing

### DISCUSSION THREAD

**Discussion by jinghua-qa** [apache/superset#18498] [Ideas]
Tooltip and column title of DATA for Time comparison with absolution difference is confusing, i think better to indicate 'difference' rather than  'absolute'

## Screenshot
<img width="1493" alt="Screen Shot 2021-07-29 at 11 39 54 PM" src="https://user-images.githubusercontent.com/81597121/127612245-2daece3c-244c-4846-b2cc-3d5354c11e59.png">

**Comment by jinghua-qa** [comment]
@zhaoyongjie

**Comment by junlincc** [comment]
Is it possible to integrate aa functionalities into metric popover? and allow user to customize those derived columns? 
should be a separate project?

**Comment by Chadys** [comment]
I was about to create a new discussion before finding this one, because the issue I have would also be solved by the proposed solution. The label used for legends is indeed veeeeery verbose and not practical at all.

When you use the "Actual values" CALCULATION TYPE, the label generated for the legend is taken directly from your dimension field value so you can customize it with a calculated or metric column:
<img width="1038" alt="Screenshot 2022-11-14 at 14 39 57" src="https://user-images.githubusercontent.com/17880877/201682480-d62f92cb-fe59-40d7-b63f-4dfc7d6005d1.png">

The label for the shifted field is a bit longer and maybe we might want to make the time shift label customizable:
<img width="1031" alt="Screenshot 2022-11-14 at 14 40 10" src="https://user-images.githubusercontent.com/17880877/201680916-d2798e56-f15a-4a02-9121-42d6e640f954.png">

But for all other CALCULATION TYPE the generated label takes too much place, making it hard to be read:
<img width="1046" alt="Screenshot 2022-11-14 at 14 41 02" src="https://user-images.githubusercontent.com/17880877/201682709-e2fa852e-949c-4866-8cdf-90eea92faf70.png">


So I am fully in favour of being able to customize it, or if there already exists a hidden way to do it I am also all ears 😄
